src/components/MiniScore.tsx

```diff
@@ -0,0 +1,17 @@
+import { getScoreColor } from "~utils/styling";
+
```

Comment:
Best practice would be to call this `MiniScoreProps `since this interface is being used exclusively to handle typescript prop types. Makes code a bit more directly readable and prevents people from misusing this interface later